common/sphinx.c

```diff
@@ -357,7 +357,7 @@ struct onionpacket *create_onionpacket(
 
 	if (!params)
 		return NULL;
-	packet->version = 1;
+	packet->version = 0;
 	memset(nexthmac, 0, SECURITY_PARAMETER);
```

Comment:
This is why we quote the spec, and why check-source checks it... would be nice to add some of that here!